governance/polls/Vault Compensation Denomination - April 13, 2020.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Vault Compensation Denomination Poll - April 13, 2020
+summary: Signal whether to provide compensation to vault holders in the vault's token (ETH/BAT) or in DAI
+discussion_link: https://forum.makerdao.com/t/1822
+poll_rules: The voter may select to vote for one of the poll options or they may elect to abstain from the poll entirely
+options:
+   0: Abstain
```

Comment:
what is the point of the abstain option? What happens if that option wins?

Reply:
A vote for the `abstain` option is considered the same as not voting, so it can't ever really "win." If it weren't there, there'd be no way to `unvote`.

Reply:
hmm, alright good enough for me